build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java

```diff
@@ -364,38 +364,16 @@ private static void configureVMWrapperTasks(
         List<ElasticsearchDistribution> currentDistros = new ArrayList<>();
 
         for (Architecture architecture : Architecture.values()) {
-            ALL_INTERNAL.stream().forEach(type -> {
-                for (boolean bundledJdk : Arrays.asList(true, false)) {
-                    if (bundledJdk == false) {
-                        // We'll never publish an ARM (aarch64) build without a bundled JDK.
-                        if (architecture == Architecture.AARCH64) {
-                            continue;
-                        }
-                        // All our Docker images include a bundled JDK so it doesn't make sense to test without one.
-                        if (type.isDocker()) {
-                            continue;
-                        }
-                    }
-                    currentDistros.add(
-                        createDistro(distributions, architecture, type, null, bundledJdk, VersionProperties.getElasticsearch())
```

Comment:
we can probably remove the bundleJdk parameter from the `createDistro`

Reply:
I purposefully didn't to make backports easier. That might be wishful thinking though.